lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -2170,11 +2320,22 @@ SwiftASTContext::CreateInstance(lldb::LanguageType language, Module &module,
 
   swift_ast_sp->SetCompilerInvocationLLDBOverrides();
 
-  // Apply the working directory to all relative paths.
-  std::vector<std::string> DeserializedArgs = swift_ast_sp->GetClangArguments();
+  // Collect search paths before importing modules.
+  const bool discover_implicit_search_paths = false;
+  const bool use_all_compiler_flags = false;
```

Comment:
If you're declaring variables for these parameters you could also declare one for `is_main_executable`